homepage/styles/main.css

```diff
@@ -0,0 +1,135 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.navbar {
+  margin-top: 10px;
+  margin-bottom: 20px;
+  margin-left: 20px;
+  margin-right: 20px;
```

Comment:
this can be written as `margin: 10px 20px 20px 20px;` or even shorter: `margin: 10px 20px 20px;`
it's a shorthand for all sides: https://developer.mozilla.org/en-US/docs/Web/CSS/margin

Works with padding too